pkg/queue/worker.go

```diff
@@ -35,7 +37,19 @@ func NewWorker(config etc.JobQueue) Worker {
 	}
 	workerPool := work.NewWorkerPool(workerContext{}, uint(config.WorkerConcurrency), config.Namespace, redisPool)
 
-	workerPool.JobWithOptions(scanArtifactJobName, work.JobOptions{Priority: 1, MaxFails: 1}, (*workerContext).ScanArtifact)
+	// Note: For each scan job a new instance of the workerContext struct is created.
```

Comment:
note: You're right. I read the source code and it looks like `ctx` is generated every time.
https://github.com/gocraft/work/blob/5959e69ad211c5ca37ffdf3ede02e35a5ae41d98/run.go#L10

Reply:
Thank you for confirming that @knqyf263 !